pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -970,3 +972,50 @@ message StatusResponse {
     // Status of the Runtime.
     RuntimeStatus status = 1;
 }
+
+message ImageFsInfoRequest {}
+
+// UInt64Value is the wrapper of uint64.
+message UInt64Value {
+    // The value.
+    uint64 value = 1;
+}
+
+// InodeInfo is the inode information of the images.
+message InodeInfo {
+    // Number of Inodes.
+    UInt64Value inodes = 7;
+    // Number of available Inodes (if known)
+    UInt64Value inodes_free = 8;
+}
+
+// FsInfo contains data about filesystem usage.
+message FsInfo {
+    // The block device name associated with the filesystem.
+    string device = 1;
+    // The root directory for the images.
+    string path = 2;
```

Comment:
I would prefer a more descriptive name here.  Path is a little generic.  I believe we generally refer to this as mountpoint in the rest of the code base.